cogboard-webapp/src/components/helpers/index.js

```diff
@@ -27,3 +27,9 @@ export const parseYupErrors = (errors) => {
   
   return result;
 }
+
+export const trimLeadingZeros = (event) => {
+  const inputValue = event.target.value;
+  const parsedValue = parseInt(inputValue);
```

Comment:
please add empty line to separate variable declarations from rest of instructions